### PR TITLE
Update user_account for Chef13

### PR DESCRIPTION
**Pull request by rearview**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license          'MIT'
 description      'Provides a resource for user_account'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '1.1.0'
+version          '1.1.1'
```
Comment (jarosser06) [inline metadata.rb]: Can you remove this version bump. Version bumps should be in master.

**Review by jarosser06 — CHANGES_REQUESTED** [review]

**Comment by rearview** [comment]
Will do.


Chris Rutledge | Systems Administrator 2
Digital Initiatives | Texas A&M University Libraries
Tel. 979.458.5588 | Fax 979.845.6238

**Review by rearview — COMMENTED** [review]
bumped version back down.

**Comment by rearview** [comment]
version bumped back down.

**Comment by jarosser06** [comment]
Looks good, can you squash these commits to a single commit.

**Comment by rearview** [comment]
I’m trying to figure out how.  This is a first for me.


Chris Rutledge | Systems Administrator 2
Digital Initiatives | Texas A&M University Libraries
Tel. 979.458.5588 | Fax 979.845.6238

**Comment by rearview** [comment]
Okay check that.


Chris Rutledge | Systems Administrator 2
Digital Initiatives | Texas A&M University Libraries
Tel. 979.458.5588 | Fax 979.845.6238

**Comment by jarosser06** [comment]
That added more commits. Try ```git rebase -i HEAD~6```

**Comment by rearview** [comment]
There you go.


Chris Rutledge | Systems Administrator 2
Digital Initiatives | Texas A&M University Libraries
Tel. 979.458.5588 | Fax 979.845.6238

**Comment by jarosser06** [comment]
I updated the commit and merged it into master.

**Pull request closed without merging**